serialisation/src/main/java/org/ehrbase/serialisation/dbencoding/wrappers/json/writer/translator_db2raw/DvCodedTextNameValue.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2020 Vitasystems GmbH and Christian Chevalley Hannover Medical School.
+ *
+ * This file is part of project EHRbase
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehrbase.serialisation.dbencoding.wrappers.json.writer.translator_db2raw;
+
+import com.google.gson.internal.LinkedTreeMap;
+import com.google.gson.stream.JsonWriter;
+import org.ehrbase.serialisation.dbencoding.wrappers.json.I_DvTypeAdapter;
+
+
+import java.io.IOException;
+import java.util.Map;
+
+public class DvCodedTextNameValue implements I_NameValueHandler{
+
+    private final JsonWriter writer;
+    private final String value;
+    private final String codeString;
+    private final String terminologyId;
+
+    DvCodedTextNameValue(JsonWriter writer, String value) {
```

Comment:
Not used nor necessary by interface. So, safe to delete?

Reply:
Yep. Done